.github/workflows/deploy-docs.yml

```diff
@@ -70,10 +70,19 @@ jobs:
         destination_dir: dev
         keep_files: false
 
-    - name: Publish to Github Pages on release
+    - name: Publish to Github Pages on release (versioned)
       if: ${{ github.event_name == 'release' }}
       uses: peaceiris/actions-gh-pages@v4
       with:
         github_token: ${{ secrets.GITHUB_TOKEN }}
         publish_dir: docs/_build/html/
         destination_dir: ${{ github.ref_name }}
+
+    - name: Publish to Github Pages on release (latest)
+      if: ${{ github.event_name == 'release' }}
+      uses: peaceiris/actions-gh-pages@v4
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: docs/_build/html/
+        destination_dir: latest
+        keep_files: false
```

Comment:
If I read this correctly, these 2 lines will "re-deploy" to _latest_ every time this executes, thus, having the latest docs always available in that dir. Nice and simple. Makes sense to me. :relaxed: